edx_proctoring/templates/emails/proctoring_attempt_status_email.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+
+{% blocktrans %}
+
+This email is to let you know that the status of your proctoring session review for {{ exam_name }} in
+<a href="{{ course_url }}">{{ course_name }} </a> is {{ status }}. If you have any questions about proctoring,
+contact edX support at info@edx.org.
```

Comment:
@chrisndodge I couldn't info@edx.org in any config file. Can you please look into it? Thanks